content/en/apps/guides/integrations/best-practices/rapidpro.md

```diff
@@ -4,106 +4,105 @@ linkTitle: "RapidPro"
-
-### App settings
-
-Update your app settings as follows.
-
+### Introduction
+[RapidPro](https://app.rapidpro.io/) is the Open Source version of Textit, developed by UNICEF and Nyaruka, to  visually build workflow logic targeting mobile-based services. Here, we detail the best practices when working on a RapidPro deployment. Top of the list is to review RapidPro’s documentation to familiarize with various components including the [API](https://rapidpro.io/api/v2/). Next, we focus on various components or stages in the deployment process.
+
+### [Workspaces](https://rapidpro.github.io/rapidpro/docs/orgs/)
+Think of workspaces as the equivalent of an instance in the CHT. Medic has an internally hosted RapidPro. Ensure that you request for both `staging` and `production` workspaces early enough to allow SRE set up the workspaces on time.
+
+### SMS Messaging Channels
+- RapidPro supports both Android channels and SMS aggregators. Note that Android channels have a messaging limit of 330 outgoing messages/hour. If using Android channels, ensure that you install the maximum number of SMS packs and follow the following best practices for best results:
+   1. Avoid the Android Channel going to sleep 
+    The following apply (based on Samsung A7 2018) with more details available on [dontkillmyapp](https://dontkillmyapp.com/samsung):
+      - Android Settings > Apps > RapidPro > Battery > Optimize battery usage > Don't Optimize.
+      - Android Settings > Device care > Battery > App power management > Apps that won’t be put to sleep. Make sure that the app is in this list. Turn off "Put unused apps to sleep."
+      - Android Settings > System > Developer Options. Background Limit is set to Standard Limit. Make sure the RapidPro app has the toggle enabled in "Background Check" and ALL of my apps have "App Standby state: ACTIVE" in Standby apps.
+      - Keep the phone plugged into a power source.
+
+   2. Automatically Wake up the Phone 
+      - The production RapidPro system uses Google Cloud Messaging which wakes up the phone whenever a message is sent. Install an APK that makes the channel to relay messages as soon as RapidPro emits.
+   3. Alerts for when things go wrong 
+      - Send monitoring emails and alert various parties when the Android Channel goes to sleep or becomes unavailable. This can be done from the channel's management page under `Alert Email`
+
+- The Android channels can however, be used with a [bulk sender](https://help.nyaruka.com/en/articles/5153032-using-a-bulk-sender) to get past the 330 outgoing messages per hour.
+- We recommend that you use shortcodes for SMS messaging, which can either be shared or dedicated. Dedicated are preferred because recipients do not have to include the keyword with each response submitted. Make sure to make arrangements for the shortcode in advance to avoid lags due to the duration it might take to fully set up one.
```

Comment:
```suggestion
- We recommend that you use shortcodes for SMS messaging, which can either be shared or dedicated. Dedicated Shortcodes are preferred because recipients do not have to include the keyword with each response submitted. Shortcode procurement can be a lengthy proceses, so make sure to make arrangements for the shortcode in advance. It is possible but inconvenient to migrate to a shortcode after deployment.
```